examples/CommandBasedRobot/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -81,4 +141,24 @@ public void periodic() {
   public void simulationPeriodic() {
     // This method will be called once per scheduler run during simulation
   }
+
+  public double applyDeadband(double value) {
+    if (Math.abs(value) < OIConstants.JOYSTICK_DEADBAND) {
+      return 0;
+    }
+    else {
+      return value;
+    }
+  }
+
+  /**
+   * Calculates the angle of translation set by the left stick.
+   *
+   * @return The angle of translation, in degrees. 0 corresponds to forwards, and positive
+   *     corresponds to counterclockwise.
+   */
+  private double calculateTranslationDirection(double x, double y) {
+    // Calculate the angle.
+    return Math.toDegrees(Math.atan2(y, x));
```

Comment:
I found a bug with this in testing

```suggestion
    // Swapping x/y and inverting y because our coordinate system has +x forwards and -y right
    return Math.toDegrees(Math.atan2(x, -y));
```